### PR TITLE
More ghost dojo tweaks

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

Adds a button to the crematorium (whoops), makes ghost dojo peeps pacifists, adds an autoylathe.

## Why It's Good For The Game

Most of these were suggested. The dang crematorium was a dumb mistake.

## Changelog
:cl:
add: Ghost dojo now has an autoylathe.
del: Ghost dojo no longer has VR or modular computer access.
tweak: Ghost dojo mobs are pacifists.
tweak: Ghost dojo booze-o-mat is now all access.
fix: Ghost dojo crematorium now has a button.
/:cl:

### CODE REVIEW

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -16736,6 +16715,10 @@
 	smooth = 1
 	},
 /area/centcom/holding)
+"Nf" = (
+/obj/machinery/autoylathe,
```
Comment (Trilbyspaceclone) [inline _maps/map_files/generic/CentCom.dmm:16719]: Dont forget to add in plastic or a chem dispender to make it.
Also can use a new stack of glass/metal/wood

Comment (Ghommie) [reply _maps/map_files/generic/CentCom.dmm:16719]: Otherwise make a preloaded subtype, whatever works.

**Review by Ghommie — APPROVED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -619,6 +619,24 @@
...
+	icon_icon = 'icons/mob/mob.dmi'
+	button_icon_state = "ghost"
+	name = "Toggle deadchat"
+	desc = "Turn off or on your ability to hear ghosts."
+
+/datum/action/toggle_dead_chat_mob/Trigger()
+	if(!..())
+		return 0
+	var/mob/M = target
+	if(HAS_TRAIT(M,TRAIT_SIXTHSENSE))
```
Comment (Ghommie) [inline code/game/objects/structures/ghost_role_spawners.dm]:
```suggestion
	if(HAS_TRAIT_FROM(M,TRAIT_SIXTHSENSE,GHOSTROLE_TRAIT))
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Requested change above.

**Comment by Ghommie** [comment]
That binary borg issue, that can be fixed in another PR but uh uh :eyes:

**Review by Ghommie — APPROVED** [review]

**Pull request merged**